docs/usage.md

```diff
@@ -0,0 +1,240 @@
+| [Home](https://github.com/fortinet-fortisoar/solution-pack-connector-health-monitoring/blob/release/1.0.0/README.md) |
+|----------------------------------------------------------------------------------------------------------------------|
+
+# Usage
+
+The Connector Health Monitoring solution pack notifies users of specified connectors' health. The recipients receive all connectors' statuses in a tabular form, detailing each version and configuration available for the listed connectors. The solution pack reports the connectors' health based on an *Include* and *Exclude* list.
+
+Following section describes the format in which to declare the Exclude and Include list, and the field location on the user interface(UI) where you can specify it.
+
+## Include and Exclude List Format
+
+Following is the format in which to specify the exclude and include list of connectors. Each connector is listed as an object, separated by a comma in an array.
+
+```JSON
+[
+  {
+    "name": "",
+    "version": "",
+    "configName": []
+  }
+]
+```
+
+For example, following snippet shows how to declare multiple connectors for exclude or include list:
+
+```JSON
+[
+  {
+    "name": "VirusTotal",
+    "version": "3.0.0",
+    "configName": [ "vt-1" ]
+  },
+  {
+    "name": "Exchange",
+    "version": "4.0.0",
+    "configName": [ "demo-1" ]
+  }
+]
+```
+
+| Field Type | Key Value Pair                         | Description                                                                                                                                                           |
+|:-----------|:---------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **string** | `"name": "ConnectorName"`              | This is a *mandatory* field. If left blank, the connector appears under **Invalid** when specified under *Include* list and ignored when specified in *Exclude* list. |
+| **string** | `"version": "VersionNumber"`           | This is an *optional* field. If left blank, it fetches status of all available versions                                                                               |
```

Comment:
This denotes Connector version. This is an optional field.........

Same can be done for other fields as well.

Reply:
Can we give Version example as well here like 2.0.0 and not v2.0.0 as it's mentioned in docs like this.